alert-common/src/main/java/com/synopsys/integration/alert/common/action/ValidationActionResult.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.action;
+
+import java.util.List;
+
+import org.springframework.http.HttpStatus;
+
+import com.synopsys.integration.alert.common.descriptor.config.field.errors.AlertFieldStatus;
+
+public class ValidationActionResult extends ActionResult<Void> {
+    private List<AlertFieldStatus> fieldStatusList;
```

Comment:
Why isn't this the content of `ActionResult`? It would probably be better to update `ValidationResponseModel` and make this `ActionResult<ValidationResponseModel>` instead.